superset-frontend/src/views/CRUD/alert/ExecutionLog.tsx

```diff
@@ -165,7 +165,14 @@ function ExecutionLog({ addDangerToast, isReportEnabled }: ExecutionLogProps) {
         name={
           <StyledHeader>
             <span>
-              {alertResource?.type} {alertResource?.name}
+              {alertResource
+                ? alertResource.type === 'Alert'
+                  ? t('Alert:')
+                  : alertResource.type === 'Report'
+                  ? t('Report:')
```

Comment:
No strong feeling here, but I wonder if we might leverage existing translations by doing this as
```suggestion
                ? alertResource.type === 'Alert'
                  ? `${t('Alert')}:`
                  : alertResource.type === 'Report'
                  ? `${t('Report:')}:`
```
¯\\\_(ツ)_/¯